src/js/components/Field/Field.tsx

```diff
@@ -19,72 +26,113 @@ import { FieldLabel } from "../FieldLabel";
 import { FieldMessage } from "../FieldMessage";
 import { Input } from "../Input";
 import { Status, Size } from "../../types";
+import { Checkbox } from "../Checkbox";
+import { CheckboxState, CheckboxStates } from "../../types";
+
+const checkboxShape = PropTypes.shape({
```

Comment:
I dont think you'll want more than one checkbox in a field. I think the recommended approach will be to use multiple fields. I can add a storybook example for that and make sure it works as expected.

Reply:
I was thinking of a "Check all that apply" type scenario, where you would want one message and one label per group:
<img width="370" alt="Screen Shot 2020-12-08 at 8 51 26 AM" src="https://user-images.githubusercontent.com/9888467/101499320-bef7f000-3932-11eb-90dd-b6b3aea66268.png">

And would want this for sure for radios.

Reply:
Pushed up an example that uses multiple checkboxes and manual Field.Label and Field.Caption usage to indicate how this could be accomplished. We may revisit this once we tackle radios, as we may find a better way to handle this while building them out.

Reply:
Cool, that seems fine for now. I imagine we'll want it integrated in the future to handle required fields, error messages, etc, but we can tackle that when we get to it.

Reply:
I really want to be cautious there as I dont think our component library should dictate how a user does form validation. My goal, especially for form fields, is to expose whats needed so that people can wire up whatever they want to work with these fields.